File: Sistema_Inventarios_API/Sistema_Inventarios_MOD/Venta.cs
using System;
using System.Collections.Generic;

namespace Sistema_Inventarios_MOD;

public partial class Venta
{
    public int IdVenta { get; set; }

    public string? NumeroDocumento { get; set; }

    public string? NombreCliente { get; set; }

    public string? TipoPago { get; set; }

    public decimal? Total { get; set; }

    public DateTime? FechaRegistro { get; set; }

    public virtual ICollection<DetalleVenta> DetalleVenta { get; set; } = new List<DetalleVenta>();
}

File: Sistema_Inventarios_APP/src/app/interfaces/venta.ts
import { DetalleVenta } from "./detalle-venta";

export interface Venta {
    idVenta?: number,
    numeroDocumento?: string,
    nombreCliente: string,
    tipoPago: string,
    totalTxt: string,
    fechaRegistro?: string,
    detalleVenta: DetalleVenta[]
}
